client/state/themes/selectors.js

```diff
@@ -574,6 +574,18 @@ export function isThemePremium( state, themeId ) {
 }
 
 /**
+ * Whether a WPCOM theme given by its ID belongs to the Premium Squared bundle.
+ *
+ * @param  {Object} state   Global state tree
+ * @param  {Object} themeId Theme ID
+ * @return {Boolean}        True if the theme is in the Premium Squared bundle
+ */
+export function isPremiumSquaredTheme( state, themeId ) {
+	const theme = getTheme( state, 'wpcom', themeId );
+	return theme && includes( PREMIUM_SQUARED_THEMES, theme.stylesheet );
```

Comment:
BTW we try to cover our Redux state tree by tests as much as possible, so could you write tests for this new selector in `state/themes/test/selectors`? Check that file's contents for inspiration and feel free to ping me if you have any questions.